docs/user/customization.md

```diff
@@ -70,6 +70,25 @@ push down, editor.action.moveLinesDownAction
 
 Now when you say eg "push down air and bat", cursorless will first select the two tokens with a gray hat over the `a` and `b`, then issue the VSCode command `editor.action.moveLinesDownAction`, and then restore your original selection.
 
+## \[Experimental\] Cursorless custom regex scopes
+
+You can define custom scopes in `user/cursorless-settings/experimental/regex_scope_types.csv`
+
+> [!TIP]
+> Use `visualize [scope]` to see a live preview of the regex scope (must save `regex_scope_types.csv` to see changes)
+
+---
+Command `visualize dotted` with the following scope type `dotted,"[\w.]+"`
+
+![image](https://github.com/cursorless-dev/cursorless/assets/13824365/b54263a3-74b7-40d5-a1aa-cd7bbe86430b)
```

Comment:
huh what is this assets thing? is this because it was on the wiki? I wonder if it will stick around once it is removed from the wiki 🤔. We could just put it in assets in the repo

Reply:
That's where it was put when I attached the screenshot to the wiki page.

Reply:
When you say "just put it in assets in the repo", what directory are you referring to?

Reply:
See https://github.com/cursorless-dev/cursorless/blob/cursorless-tutorial/docs/user/README.md?plain=1#L188